client/my-sites/menus/menu-editable-item.jsx

```diff
@@ -296,28 +305,35 @@ var MenuEditableItem = React.createClass( {
 		return [
 			{
 				key: 'remove',
-				className: 'noticon noticon-trash',
+        title: this.translate( 'Delete' ),
+        className: 'is-scary',
 				showIfNew: false,
+        icon: 'trash',
 				onClick: this.remove
 			},
 			{
 				key: 'move',
-				label: this.translate( 'Move', { textOnly: true } ),
+        label: this.translate( 'Move' ),
```

Comment:
We need to keep the `{ textOnly: true }` option, otherwise the translator code will wrap the label in an element.